resources/home/dnanexus/utils/io.py

```
@@ -0,0 +1,366 @@
+        if cell[0].value:
+            if re.match(r"X[\d]+", cell[0].value):
+                sheet[cell[0].coordinate].font = Font(
+                    bold=True, name=DEFAULT_FONT.name
+                )
+            elif re.match(
+                r"[a-zA-Z0-9]+-[a-zA-Z0-9]+-[a-zA-Z0-9]+-[a-zA-Z0-9]+-[MFU]-[a-zA-Z0-9]+",
+                cell[0].value,
+            ):
+                sheet[cell[0].coordinate].font = Font(
+                    bold=True, name=DEFAULT_FONT.name
+                )
```

Ruff (0.8.0)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)